rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2846,6 +2846,13 @@ public BigInteger registerFlyoverBtcTransaction(
         return co.rsk.core.Coin.fromBitcoin(totalAmount).asBigInteger();
     }
 
+    public Optional<Keccak256> getPegoutCreationRskTxHashByBtcTxHash(Sha256Hash btcTxHash) {
+        if (activations.isActive(ConsensusRule.RSKIP298)) {
+            return provider.getPegoutCreationRskTxHashByBtcTxHash(btcTxHash);
+        }
+        return Optional.empty();
```

Comment:
```suggestion
            return activations.isActive(ConsensusRule.RSKIP298) ? provider.getPegoutCreationRskTxHashByBtcTxHash(btcTxHash).orElse(ByteUtil.EMPTY_BYTE_ARRAY) : ByteUtil.EMPTY_BYTE_ARRAY
```
This way BridgeSupport has the responsability of handling the response from the provider. Bridge class simply calls and returns

Reply:
I see. The other methods are implemented in that way. I was doubtful about that, but now it's clear.